{{cookiecutter.repostory_name}}/deploy.sh

```diff
@@ -27,6 +27,11 @@ docker-compose run --rm app sh -c "python manage.py wait_for_database --timeout
 # start everything
 docker-compose up -d
 
+{% if cookiecutter.use_slack_notification %}
+# Send slack notification about deploy
+docker-compose run --rm app sh -c "python bin/notify.py"
```

Comment:
Not sure how working with that template looks like in real but **deploy-to-aws** (which is called by github action) doesn't call **deploy** (or I don't see something). If so, the problem where to call it stands.

Reply:
What solution do you suggest?

Reply:
I see two options.
1) Run it [here](https://github.com/reef-technologies/cookiecutter-rt-django/blob/master/%7B%7Bcookiecutter.repostory_name%7D%7D/devops/scripts/deploy-backend.sh#L10). Technically we have container built there so could run script. Currently we do not COPY **bin** directory to container but I guess it's not intended.
2) Technically we could add it [here](https://github.com/reef-technologies/cookiecutter-rt-django/blob/master/%7B%7Bcookiecutter.repostory_name%7D%7D/app/envs/prod/entrypoint.sh#L7) but in case of restarting container we will get notifications.